web/crux-ui/locales/en/deployments.json

```diff
@@ -3,5 +3,6 @@
   "deploysNameDeploy": "Products - {{product}} - Versions - {{version}} - Deployments - {{name}} - Deploy",
   "log": "Log",
   "environmentConfig": "Environment config",
-  "created": "Created"
+  "created": "Created",
+  "noItems": "You haven't made a deployment. To add a deployment, click 'Deploy'."
```

Comment:
I don't think there's a deploy button on the deployments page.
![image](https://user-images.githubusercontent.com/3697447/188073660-0df35e77-4322-43e2-8c7f-d30a2deae0c6.png)